boefjes/boefjes/plugins/kat_kat_finding_types/kat_finding_types.json

```diff
@@ -70,479 +70,402 @@
+        "recommendation": "Adjust the CSP header to make it as strict as possible in order to reduce the attack surface."
     },
     "KAT-POTENTIAL-MALWARE": {
         "description": "This software is known to be used as malware.",
-        "source": null,
+        "source": "https://learn.microsoft.com/en-us/microsoft-365/security/defender/criteria?view=o365-worldwide",
         "risk": "high",
-        "impact": null,
-        "recommendation": null
+        "impact": "Depending on the malware this could be anything from eavesdropping to a ransomware attack.",
+        "recommendation": "Alert your CERT to verify if your systems are compromised."
     },
     "KAT-EXPOSED-SOFTWARE": {
-        "description": "This software should not be exposed to the Internet.",
-        "source": null,
+        "description": "This software is often used for administrative purposes and should not be exposed to the Internet. This could be an easy entry point into your organisations environment.",
+        "source": "Check your OpenKAT install on what software was identified.",
         "risk": "critical",
-        "impact": null,
-        "recommendation": null
+        "impact": "Impact depends on the identified software.",
+        "recommendation": "Move the software to a more secure location and/or make it only accessible through a VPN. "
     },
     "KAT-VERIFIED-VULNERABILITY": {
-        "description": "A verified vulnerability is found.",
-        "source": null,
+        "description": "A verified vulnerability is found by BinaryEdge.",
+        "source": "Check your OpenKAT install on what software was identified.",
         "risk": "critical",
-        "impact": null,
-        "recommendation": null
-    },
-    "KAT-INVALID-SPF-SYNTAX": {
-        "description": "SPF record does not have valid syntax.",
-        "source": "http://www.open-spf.org/SPF_Record_Syntax/",
-        "risk": "high",
-        "impact": null,
-        "recommendation": null
-    },
-    "KAT-612": {
-        "description": "Parsing an SPF record should require no more than 10 DNS lookups.",
-        "source": null,
-        "risk": "medium",
-        "impact": null,
-        "recommendation": null
-    },
-    "KAT-SPF-TOO-LONG": {
-        "description": "SPF record exceeds allowed length of 255 characters.",
-        "source": null,
-        "risk": "medium",
-        "impact": null,
-        "recommendation": null
-    },
-    "KAT-INVALID-IP-IN-SPF": {
-        "description": "An invalid ip address was found in the SPF record.",
-        "source": null,
-        "risk": "low",
-        "impact": null,
-        "recommendation": null
-    },
-    "KAT-SPF-REDIRECT-LOOP": {
-        "description": "The SPF record contains a redirect loop.",
-        "source": null,
-        "risk": "medium",
-        "impact": null,
-        "recommendation": null
-    },
-    "KAT-616": {
```

Comment:
There's no possibility of a regression removing old entries or some migration needed? CC @noamblitz

Reply:
The next kat_findings boefje run that looks at this file will update any changed findings. Any deleted records in this file would just end up as empty findings in the ooi list (the ID and finding itself would still remain)